Description: Switch to Python3
 Change shebang to use python3
Author: Sophie Brun <sophie@offensive-security.com>
Last-Update: 2018-12-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/discovery/shodan/cli/worldmap.py
+++ b/discovery/shodan/cli/worldmap.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 '''
 F-Secure Virus World Map console edition
 
--- a/lib/hostchecker.py
+++ b/lib/hostchecker.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # encoding: utf-8
 """
 Created by laramies on 2008-08-21.
--- a/theHarvester.py
+++ b/theHarvester.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 import sys
 import os
